### PR TITLE
Set current release links to 1.2

### PR DESCRIPTION
**Pull request by ararslan**

### CODE REVIEW

**Comment by fredrikekre** [comment]
Should 1.1 be moved to older releases as unmaintained?

**Comment by ararslan** [comment]
Yeah, good idea

**Comment by ararslan** [comment]
Done

**Comment by fredrikekre** [comment]
Links on the frontpage and in the footer too? :)

File: index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: homepage
 title:  The Julia Language
-julia_version: v1.1
+julia_version: v1.2
```
Comment (ararslan) [inline index.html]: > Links on the frontpage and in the footer too? :)

Those are taken care of by this change

**Comment by ararslan** [comment]
Thanks for the review, @fredrikekre ❤️

**Comment by ViralBShah** [comment]
Would be nice to have variables in github pages for the various release versions, so that we don't have to update them manually.

**Pull request merged**